data_structures/test/sqlist_test.c

```diff
@@ -0,0 +1,129 @@
+/**
+ * @file
+ * @brief Test for sqlist.c
+ * @details Test for insert, delete, search, locate operations of sqlist
+ * @author danielhu19 (shady030314@gmail.com)
```

Comment:
Remember that Markdown formatting can be used. 🙂
```suggestion
 * @author [Daniel Hu](https://github.com/danielhu19)
```